js/ui/components/clubhouse.js

```diff
@@ -120,7 +121,10 @@ var ClubhouseButtonManager = new Lang.Class({
     _init: function() {
         this._openButton =
             new St.Button({ child: new St.Icon({ style_class: 'clubhouse-open-button-icon' }) });
-        this._openButton.connect('clicked', () => { this.emit('open-clubhouse'); })
+        this._openButton.connect('clicked', () => {
```

Comment:
We may want to stop the pulsating right on `hover`, but this is just my impression. We may keep it like this for now.